pkg/api/persistentvolume/util.go

```diff
@@ -32,6 +33,15 @@ func DropDisabledFields(pvSpec *api.PersistentVolumeSpec, oldPVSpec *api.Persist
 	}
 }
 
+// AddDisabledFieldsTo to the provided set of resetFields
+// This should be called with the resetFields for all resources containing a Pod
```

Comment:
Is this really about pod objects?

Reply:
I think that's an unfortunate copy-paste :-)

Reply:
Nope, good catch. Likely a copy/pasta carryover.